cli/cfncluster/cfncluster.py

```diff
@@ -99,19 +98,26 @@ def create(args):
 
             if status != 'CREATE_COMPLETE':
                 logger.critical('\nCluster creation failed.  Failed events:')
-                events = cfnconn.describe_stack_events(stack)
+                events = cfn.describe_stack_events(StackName=stack_name).get('StackEvents')
                 for event in events:
-                    if (event.resource_status == 'CREATE_FAILED'):
+                    if event.get('ResourceStatus') == 'CREATE_FAILED':
                         logger.info("  - %s %s %s" %
-                                    (event.resource_type, event.logical_resource_id,
-                                     event.resource_status_reason))
+                                    (event.get('ResourceType'), event.get('LogicalResourceId'),
+                                     event.get('ResourceStatusReason')))
             logger.info('')
-            outputs = cfnconn.describe_stacks(stack)[0].outputs
+            outputs = cfn.describe_stacks(StackName=stack_name).get("Stacks")[0].get('Outputs', [])
             for output in outputs:
-                logger.info(output)
+                logger.info("%s : %s" % (output.get('OutputKey'), output.get('OutputValue')))
         else:
-            status = cfnconn.describe_stacks(stack)[0].stack_status
+            status = cfn.describe_stacks(StackName=stack_name).get("Stacks")[0].get('StackStatus')
             logger.info('Status: %s' % status)
+    except ClientError as e:
+        if e.response.get('Error').get('Code') == 'AlreadyExistsException':
```

Comment:
we are not catching all the other error codes, like 'ValidationError' in the case cluster name is not valid (e.g containing a '_')
I think we should just output the error message and remove the else, as it is ugly to have the stack trace displayed in this case
```
except ClientError as e:
            logger.critical(e.response.get('Error').get('Message'))
            sys.stdout.flush()
            sys.exit(1)
```